src/Models/User.php

```diff
@@ -21,25 +23,40 @@
 use Tipoff\Support\Models\BaseModel;
 use Tipoff\Support\Traits\HasPackageFactory;
 
-class User extends BaseModel implements UserInterface, ChargeableInterface
+/**
+ * @property int id
+ * @property string first_name
+ * @property string last_name
+ * @property string username
+ * @property string password
+ * @property string remember_token
+ * @property string bio
+ * @property string title
+ * @property string provider_name
+ * @property string provider_id
+ * @property string email
+ * @property Carbon|null email_verified_at
+ * @property Collection email_addresses
+ * @property Carbon deleted_at
+ * @property Carbon created_at
+ * @property Carbon updated_at
+ */
+class User extends BaseModel implements UserInterface, CanResetPasswordContract, ChargeableInterface
 {
     use HasPackageFactory;
     use SoftDeletes;
     use Authenticatable;
     use Authorizable;
     use CanResetPassword;
-    use MustVerifyEmail;
+    use MustVerifyEmail;        // TODO - whoever is handling email verification needs to review inclusion of this trait
```

Comment:
@drewroberts - note this.  Who ever is working on email validation needs to determine what to do with this trait - either significant overrides of the methods it provides or removal because its no longer relevant to how email address verification takes place.

Reply:
Ok, let's remove it. We'll want to take away the user registration page. Users will only be able to be registered through the checkout process and via Nova.